### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by elena-sek**
Created R package "ProductivityApp" which contains 3 functions so far:` create_empty_task_df, add_task, generate_eisenhower_matrix`. These functions also include descriptions.

### CODE REVIEW

**Comment by jessiwitti** [comment]
In general the basic structure of the R package is there + there is documentation about every function. I really like your first functions, it seems straight forward and works fine. A small suggestion: Maybe you can add to your create empty task description that no argument is needed (and maybe even does not work with an argument). Good work! :)

File: ProductivityApp/R/ProductivityApp_functions.R [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  )
+  if (shiny == TRUE){
+    df(rbind(df(), new_row))
+  } else {
+    rbind(df, new_row)
+  }
+}
+
+#' @title Generate an Eisenhower Matrix
+#' @description Creates an Eisenhower matrix based on a data frame with tasks.
```
Comment (brunykrijgsman) [inline ProductivityApp/R/ProductivityApp_functions.R]: Suggestion: Make it clearer what an Eisenhower matrix represents in the discription.

File: ProductivityApp/R/ProductivityApp_functions.R [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    Importance = numeric(),
+    Urgency = numeric(),
+    Enjoyability = numeric(),
+    stringsAsFactors = FALSE
+  )
+}
+
+#' @title Add Task 
+#' @description Adds a new task with its variables to the task data frame with columns Task, Duration, Importance, Urgency, and Enjoyability.
+#' @param df name of the task data frame that will be updated
```
Comment (brunykrijgsman) [inline ProductivityApp/R/ProductivityApp_functions.R:29]: Suggestion: To make it more consistent, add the parameter type to this df parameter too.

**Review by brunykrijgsman — APPROVED** [review]
It's a super cool project! 
- Everything is included in the R package skeleton.
- The documentation of the functions is clear (I made some suggestions in the code).

**Review by akane7011 — COMMENTED** [review]
Looks very clear and well-documented to me! I suppose this is for prioritizing some stuff over others, so maybe it would also be nice to color things that should be prioritized in the plot.

**Pull request merged**